sdk/textanalytics/ai-text-analytics/CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Release History
 
+## 1.0.0-preview.2 (2020-02-11)
+
+- Overhauled Text Analytics samples packages. #7052
+- Added support for rotating API keys via the `TextAnalyticsApiKeyCredential.updateKey` method. #6972
+- Added a discriminant type for `TextAnalyticsResult.error` to allow for easy differentiation between success and error types using `if (result.error) { ... }` in both TypeScript and JavaScript. #7046
+- [Breaking] Added new return types `CategorizedEntity` and `PiiEntity` for the `recognizeEntities` and `recognizePiiEntities` methods respectively. These types are equivalent to the previous `Entity` type. #7220
+- [Breaking] Removed the `detectedLanguages` property from `DetectLanguageResult`, as the service currently only returns one language. #7220
+- [Breaking] Renamed `detectLanguages` to `detectLanguage`. #7220
+- [Breaking] Renamed `documentScores` and `sentenceScores` on the `DocumentSentiment` and `SentenceSentiment` types both to `sentimentScores`. #7220
+- [Breaking] Renamed `type` and `subtype` to `category` and `subcategory` respectively in the `CategorizedEntity` and `PiiEntity` types. #7220
+- [Breaking] Renamed `DocumentSentimentValue` and `SentenceSentimentValue` to `DocumentSentimentLabel` and `SentenceSentimentLabel` respectively. #7220
+- [Breaking] Renamed `SentimentConfidenceScorePerLabel` to `SentimentScorePerLabel`. #7220
+- Migrated to TypeScript 3.7. #7210
```

Comment:
Does migration to 3.7 has any user facing impact? If not maybe less interesting to list here.